### PR TITLE
SHARD-35: rename mustUseAdminCert + add adminCertEnabled check when validating …

### PR DESCRIPTION
**Pull request by ahmxdiqbal**
…GT nodes + call out possible redundant code

### CODE REVIEW

**Review by aniketdivekar — DISMISSED** [review]

**Review by urnotsam — DISMISSED** [review]

**Review by mgthuramoemyint — DISMISSED** [review]

**Comment by ahmxdiqbal** [comment]
PR has been tested, can be merged after approvals

**Review by urnotsam — APPROVED** [review]

**Pull request merged**
